### ISSUE TITLE
Maybe missing files?

### ISSUE THREAD

**Issue by Yukang-Lin** [Some-random/theorem-proving-reasoning#2]
Hi, thanks for your wonderful work. I have met several problems trying to reproduce your experiment, hopefully you could help me with that. 
In step 2, I didn't find the reprover_train_FOLIO folder. And what's the operation in test_dataset_construction.py since I cannot find that either. I think the operation is likely to generate-benchmark-lean4.ipynb.
In step 3, I didn't find the relative config files like cli_random_text_math_FOLIO_concise.yaml..., is that identical to the original ReProver project config?

**Comment by Yukang-Lin** [comment]
BTW, I think "100 theorem proofs for ProofWriter and 27 theorem proofs for FOLIO" in paper section 4.2 referring to data_for_share folder. But it is not very clear about test set described "This gave us 99 test examples for ProofWriter and 28 for FOLIO. " in section 5.2.

**Comment by Some-random** [comment]
Thanks for your interest! Let me get to your questions one by one:

> didn't find the reprover_train_FOLIO folder

You just have to create this folder and then put the training data into it

> what's the operation in test_dataset_construction.py

As mentioned [here](https://github.com/Some-random/theorem-proving-reasoning/issues/1), the content of test_dataset_construction is unfortunately missing. But now that I think about this more, you probably don't need this function actually because the training data has already been generated

> missing config files

Thanks for spotting that! And yes its content is basically the same as the one used in the original ReProver project

> numbers in 5.2

As shown in Table 1, we manually annotated 100 problems for ProofWriter and 40 problems for FOLIO. We then use the ones that have been successfully formalized by GPT-4 to test LeanReasoner. This resulted in 100 * 99% = 99 problems for ProofWriter and 40 * 70% = 28 problems for FOLIO. 

The 100 proofs for ProofWriter and 27 proofs for FOLIO (section 4.2) are the training data we annotated.
